<?xml version="1.0"?>
<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/task2/waving_person/camera_info"/>
    <remap from="image_rect" to="/task2/waving_person/depth_mask"/>
    <remap from="points" to="/task2/waving_person/camera/depth/points"/>
  </node>
</launch>